Add description on `admin pause ddl jobs` and `admin resume ddl jobs` (#13897)
pingcap/docs: sql-statements/sql-statement-admin-resume-ddl.md

---
title: ADMIN RESUME DDL
summary: An overview of the usage of ADMIN RESUME DDL for the TiDB database.
---

# ADMIN RESUME DDL

> `ADMIN RESUME DDL` allows you to resume a paused DDL job. The `job_id` can be found by running [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md).

Reviewer: ```suggestion
`ADMIN RESUME DDL` allows you to resume a paused DDL job. You can find the `job_id` by running [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md).
```

You can use this statement to resume a paused DDL job. After the resume is complete, the SQL statement that executes the DDL job continues to show as being executed. If you try to resume a DDL job that has already been completed, you will see the `DDL Job:90 not found` error in the `RESULT` column, which indicates that the job has been removed from the DDL waiting queue.

> 

Reviewer: ```suggestion

> **Warning:**
>
> This feature is an experimental feature. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.

```

---

> title: ADMIN RESUME DDL
> summary: An overview of the usage of ADMIN RESUME DDL for the TiDB database.
> ---
> 
> # ADMIN RESUME DDL

Author: ```suggestion
title: ADMIN RESUME DDL JOBS
summary: An overview of the usage of ADMIN RESUME DDL for the TiDB database.
---

# ADMIN RESUME DDL JOBS
```

`ADMIN RESUME DDL` allows you to resume a paused DDL job. You can find the `job_id` by running [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md).

> You can use this statement to resume a paused DDL job. After the resume is complete, the SQL statement that executes the DDL job continues to show as being executed. If you try to resume a DDL job that has already been completed, you will see the `DDL Job:90 not found` error in the `RESULT` column, which indicates that the job has been removed from the DDL waiting queue.

Reviewer 2: complete ----> completed

> You can use this statement to resume a paused DDL job. After the resume is complete, the SQL statement that executes the DDL job continues to show as being executed. If you try to resume a DDL job that has already been completed, you will see the `DDL Job:90 not found` error in the `RESULT` column, which indicates that the job has been removed from the DDL waiting queue.

Author: ```suggestion
You can use this statement to resume a paused DDL job. After the resume is completed, the SQL statement that executes the DDL job continues to show as being executed. If you try to resume a DDL job that has already been completed, you will see the `DDL Job:90 not found` error in the `RESULT` column, which indicates that the job has been removed from the DDL waiting queue.
```
